test/dynamo/test_higher_order_ops.py

```diff
@@ -218,6 +218,16 @@ def f(x):
         x = torch.randn(3)
         self._test_wrap_simple(f, (x,), 3)
 
+    def test_symint_input(self):
+        def f(x):
+            i = x.size(0)
+            return wrap(lambda x, i: x.view(i), x, i)
+
+        x = torch.randn(3, 1)
+        self._test_wrap_simple(
+            f, (x,), ifdynstaticdefault(2, 3), expected_opcount=ifdynstaticdefault(1, 2)
+        )
```

Comment:
The corresponding dynamic shape version test was generated at ```test/dynamo/test_dynamic_shapes.py``` automatically, so we don't need to explicitly set ```dynamic_shape=True``` at here. And both static and dynamic shapes will be tested.